src/Tizen.NUI/src/public/BaseComponents/VideoView.cs

```diff
@@ -447,6 +449,54 @@ public string ResourceUrl
             }
         }
 
+        /// <summary>
+        /// Enables auto rotation of the video based on the orientation of the video contents.
+        /// </summary>
+        /// <note> This feature is supported only when underlay is false. </note>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AutoRotation
```

Comment:
How about adding `Is` or somthing similar for prefix? For example, `IsAutoRotationEnabled`.
I found many methods in TizenFX are using this name pattern for boolean property and also the MSDN framework design guildlines do encourage usage of `Is` or `Has`.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties
>  DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value.

Reply:
Thanks for your suggestion.